.travis.yml

```diff
@@ -25,7 +25,7 @@ before_script:
   - sh -e /etc/init.d/xvfb start
 script:
   - grunt test
-  - coverage run manage.py test
+  - manage.py test
```

Comment:
Thanks! Just figured it needs to be something like that as it is not run by coverage anymore :blush: